x-pack/plugins/session_view/public/components/detail_panel_process_tab/index.tsx

```diff
@@ -302,13 +370,21 @@ export const DetailPanelProcessTab = ({ processDetail }: DetailPanelProcessTabDe
           {
             title: <DetailPanelListItem>user.name</DetailPanelListItem>,
             description: (
-              <DetailPanelCopy textToCopy={userName}>{dataOrDash(userName)}</DetailPanelCopy>
+              <DetailPanelCopy
+                textToCopy={`${PROCESS_FIELD_PREFIX}.user.name: ${dataOrDash(userName)}`}
+              >
+                {dataOrDash(userName)}
+              </DetailPanelCopy>
             ),
           },
           {
             title: <DetailPanelListItem>group.name</DetailPanelListItem>,
             description: (
-              <DetailPanelCopy textToCopy={groupName}>{dataOrDash(groupName)}</DetailPanelCopy>
+              <DetailPanelCopy
+                textToCopy={`${PROCESS_FIELD_PREFIX}.group.name: ${dataOrDash(groupName)}`}
```

Comment:
any reason why some items weren't between `" "`? like `user.name`, `group.name`, `exit.code` and `pid`?

Reply:
I initially thought we shouldn't include `""` for `exit_code` and `pid` because they are `number`, but after looking into the copy to clipboard behaviours in other places of Kibana, I found that they include `""` for everything, I will push a fix to do that.